servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/H2ProtocolConfig.java

```diff
@@ -53,4 +54,46 @@ default String alpnId() {
      */
     @Nullable
     String frameLoggerName();
+
+    /**
+     * Configured {@link KeepAlivePolicy}.
+     *
+     * @return configured {@link KeepAlivePolicy} or {@code null} if none is configured.
+     */
+    @Nullable
```

Comment:
Since now we have a special `DISABLE_KEEP_ALIVE` constant, consider making this field not-nullable. We may start with `disabled()` as a default value and change it later if necessary.

Reply:
The constant equating to disable is just an implementation detail IMHO. This returning `null` better represents that there is no policy set.